File: 1st Year/C/Semester1/Labs/Lab9/Q4.c
/*     Program Description: This program asks the user 2 integers stores them and using pointers calculates their sum
       Author: Abraham Izarra
       Date: 25 Nov 2019
*/

#include <stdio.h>

int main()
{
    int num1, num2, sum;
    int *ptr1 = &num1;
    int *ptr2 = &num2;
    int *ptrS = &sum;
    
    
    printf ("Enter two numbers \n");
    
    scanf ("%d %d", & *(ptr1), & *(ptr2) );
    
    *ptrS = *ptr1 + *ptr2;
    
    printf ("The sum of these two numbers is %d \n", *ptrS);
    
    
    return 0;
}

File: 3rd Year/Python/Labs/Lab6/Q4.py
# sum numbers from 0 to N
Number = int(input("Enter a number: "))


# Function to sum numbers from 0 to N
def sum(number):
    total = 0

    for i in range(number + 1):
        total += i

    return total


sum(Number)

Total = sum(Number)
print(Total)
